.env.example

```diff
@@ -1,6 +1,9 @@
-DATABASE_URL=
-REDIS_URL=
+DATABASE_URL=mysql://root:root@mysql:3306/envelop
+REDIS_URL=redis://redis:6379
 SENDGRID_API_KEY=
-HOST=
-FRONT_HOST=
+HOST=http://localhost:3000
+FRONT_HOST=http://localhost:8080
 WALLET_PRIVATE_KEY=
+
+MYSQL_ROOT_PASSWORD=root
+MYSQL_DATABASE=envelop
```

Comment:
@Todmy No way to pick these from `DATABASE_URL`?

Reply:
Maybe just `MYSQL_ALLOW_EMPTY_PASSWORD` and envelop as database by default :)

Reply:
@Todmy You checked this? we don't need to update all repo's env. we can allow empty passwords and default database as it is just for local testing? for prod/test environments we can anyway use password from database url